projects/adventure/adv.py

```diff
@@ -17,20 +17,114 @@
 world.loadGraph(roomGraph)
 
 # UNCOMMENT TO VIEW MAP
-world.printRooms()
+# world.printRooms()
 
 player = Player("Name", world.startingRoom)
 
+class Queue():
+    def __init__(self):
+        self.queue = []
+    def enqueue(self, value):
+        self.queue.append(value)
+    def dequeue(self):
+        if self.size() > 0:
+            return self.queue.pop(0)
+        else:
+            return None
+    def size(self):
+        return len(self.queue)
+
+
 # Fill this out
 traversalPath = []
 
+graph = {}
+
+def backtrack(direction):
```

Comment:
nice job creating a function for this.  Similarly, we could achieve the same thing with a hash-table/dictionary.